package dev.anuradha.taskservice.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import dev.anuradha.taskservice.model.User;

import java.util.Optional;

public interface UserRepository extends JpaRepository<User, Long> {
    Optional<User> findByUsername(String username);
}
